samples/azure/shared-host/sample.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Shared Hosting in Azure Cloud Services Sample
+summary: 'Shared hosting in Azure Cloud Services.'
+tags:
+- Azure
+- Cloud Services
+- Hosting
+- Shared hosting
+- Hosting in Azure
+related:
+- nservicebus/azure/shared-hosting-in-azure-cloud-services
+---
+
+ 1. Start [Azure Storage Emulator](http://azure.microsoft.com/en-us/documentation/articles/storage-use-emulator/)
+ 1. Rebuild solution (right click on `AzureSharedHosting` solution in Solution explorer, then `Rebuild Solution`)
+ 1. Run the solution 
+ 1. Inspect Azure Storage Emulator Tables for `MultiHostedEndpointsOutput` table and its content for something like the following:
+
+| PartitionKey | RowKey | Timestamp | Message |
+|:--|:--|:--|:--|
+|Sender	|2015-05-25 14:58:33	|5/25/2015 8:58:33 PM	|Pinging Receiver |
+|Receiver	|2015-05-25 14:58:34	|5/25/2015 8:58:34 PM	|Got Ping and will reply with Pong |
+|Sender	|2015-05-25 14:58:35	|5/25/2015 8:58:35 PM	|Got Pong from Receiver |
+
+WARNING: When running sample, do not stop execution from Visual Studio, but instead stop cloud service from Azure Compute Emulator UI. Do to an issue with Azure Compute Eulator not invoking OnStop method, multi hosted endpoints are not terminated.
+
+NOTE: To inspect multi-host [role emulated file system](https://msdn.microsoft.com/en-us/library/azure/hh771389.aspx), navigate to c:\Users\<username>\AppData\Local\dftmp\Resources

```

Comment:
dftmp is a well known string and not some short guid that will be diff on diff machines?